### DISCUSSION TITLE
Problems when trying to use dosxyznrc

### DISCUSSION THREAD

**Discussion by Mr-J-ST** [nrc-cnrc/EGSnrc#1067] [Questions]
I am new to EGSnrc and wish to use dosxyznrc. However, there is a problem after installing the EGSnrc on both Windows 10 and Linux. It is unable to use dosxyznrc. When I try to use dosxyznrc, it always shows an error.
How can I fix this?

**/usr/bin/bash: -c: line 2: syntax error: unexpected end of file
process_begin: CreateProcess(NULL, echo ., ...) failed.
make (e=2): ϵͳ�Ҳ���ָ�����ļ���

mingw32-make: *** [D:\EGSnrc\HEN_HOUSE\makefiles\standard_makefile:87: D:\EGSnrc\egs_home\bin\win3264\dosxyznrc.exe] Error 2

ExitCode = 2

************************* finished *****************************
**

**Comment by rtownson** [comment]
This is probably an installation issue, so please post your installation log, found in HEN_HOUSE/logs. Do other applications compile and run?

**Comment by Mr-J-ST** [comment]
Thank you for your reply.
I've reinstalled the application on the Linux and solved the problem.
Thanks a lot.
________________________________
发件人: Reid Townson ***@***.***>
发送时间: 2023年10月16日 15:27
收件人: nrc-cnrc/EGSnrc ***@***.***>
抄送: Jiang, Shengting ***@***.***>; Author ***@***.***>
主题: Re: [nrc-cnrc/EGSnrc] Problems when trying to use dosxyznrc (Discussion #1067)


⚠ Caution: External sender


This is probably an installation issue, so please post your installation log, found in HEN_HOUSE/logs. Do other applications compile and run?
